lib/src/follower.dart

```diff
@@ -587,6 +587,15 @@ class RenderFollower extends RenderProxyBox {
     }
   }
 
+  @override
+  void markNeedsPaint() {
+    super.markNeedsPaint();
+
+    // Immediately schedule a new frame to avoid being in a dirty in the cases
```

Comment:
This generally looks good, but can we get a bit more specific with the comment? I think most readers will wonder how it's possible that we mark ourselves dirty and yet a frame isn't scheduled on our behalf.

Reply:
Updated. Do you think is it clear enough?